### ISSUE TITLE
[Bug]: Unable to update MongoDB::Atlas::Project to add teams

### ISSUE THREAD

**Issue by kichik** [mongodb/mongodbatlas-cloudformation-resources#937]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### CFN Resource version

0.2.0

### CFN Resource Region

us-east-1

### Current Behavior

After a project has been deployed, it is not possible to modify it to add teams. Adding teams at creation times works as expected.

Instead of adding a team to the project, you get the following error:

```
Resource handler returned message: "Unable to complete request: https://cloud.monn
godb.com/api/atlas/v2/groups/zzzz PATCH: HTTP 400 Bad Request (Error code: "COULD_NOT_UPDATE_PROJECT_LABEL") Detail: An error occurred when attempting to update the human-readable label that identifies the specified project. Reason: Bad Request. Params: []"
```

### CFN template to reproduce the issue

```yaml
{}
```

### Steps To Reproduce

To reproduce, deploy the following minimal template (with corrected org id):

```json
{
  "Resources": {
    "Project": {
      "Type": "MongoDB::Atlas::Project",
      "Properties": {
        "Name": "test",
        "OrgId": "xxxx"
      }
    }
  }
}
```

Then update the stack with this template (with corrected org and team id):

```json
{
  "Resources": {
    "Project": {
      "Type": "MongoDB::Atlas::Project",
      "Properties": {
        "Name": "test",
        "OrgId": "xxxx",
        "ProjectTeams": [
          {
            "TeamId": "yyyyy",
            "RoleNames": ["GROUP_OWNER"],
          }
        ]
      }
    }
  }
}
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maastha** [comment]
Hi @kichik 

Thank you for reporting this issue. I was not able to reproduce this with the latest version of MongoDB::Atlas::Project i.e. `2.2.0`.

In the issue you have mentioned version `0.2.0` which isn't a valid version.

Could you please try again with the latest version and let us know if it still doesn't work for you?

Thank you!

**Comment by kichik** [comment]
Thanks @maastha. I am no longer sure which version it was anymore. It was supposed to just be latest. I didn't specify which version to install. I got the 0.2.0 number from the releases page here as I wasn't able to find it on AWS side.

Looks like it auto-updated to 2.2.0 which was released last week (found where it's listed in AWS) and now it is working.

The relevant code seems to be the same:

https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/4ddc58f049d78fb60200994dab2b8fcf61c5a3af/cfn-resources/project/cmd/resource/resource.go#L212-L217

The only commit I can see also doesn't seem relevant:

https://github.com/mongodb/mongodbatlas-cloudformation-resources/commit/4ddc58f049d78fb60200994dab2b8fcf61c5a3af

So maybe it was just fixed on the API side? Either way, it works now 🤷‍♂️
